File: Feitos em C/1241.c
#include <stdio.h>
int main() {
	int times,i,j;
	scanf("%d",&times);
	for(i=0;i<times;i++){
		char a[1000],b[1000];
		for(j=0;j<1000;j++){
			a[j]=65;
			b[j]=65;
		}
		scanf("%s %s",&a,&b);
		int sizeA = strlen(a), sizeB = strlen(b),cont=0,auxB=sizeB;
		for(j=sizeA-1;j>=(sizeA-1)-(sizeB-1);j--){
			if(a[j]==b[auxB-1]){
				cont++;
			}
			auxB--;
		}
		if(cont==strlen(b)){
			printf("encaixa\n");
		}
		else{
			printf("nao encaixa\n");
		}
	}
    return 0;
}

File: Feitos em C Sharp/1241.cs
using System;

namespace ConsoleApp5 {
    class Program {
        static void Main(string[] args) {
			int times =int.Parse(Console.ReadLine());
			for (int i = 0; i < times; i++) {
				string[] phrase = Console.ReadLine().Split();
				string a = phrase[0];
				string b = phrase[1];
				int sizeA = a.Length; 
				int sizeB = b.Length;
				string auxB;
				if (sizeB > sizeA) {
					Console.WriteLine("nao encaixa");
					continue;
				}
				else {
					auxB = a.Substring((sizeA - 1) - (sizeB - 1));
				}
				if (auxB==b) {
					Console.WriteLine("encaixa");
				}
				else {
					Console.WriteLine("nao encaixa");
				}
			}
		}
    }
}
